shell/browser/browser_linux.cc

```diff
@@ -25,11 +29,19 @@
 
 namespace electron {
 
+const char kFreedesktopPortalName[] = "org.freedesktop.portal.Desktop";
+const char kFreedesktopPortalPath[] = "/org/freedesktop/portal/desktop";
+const char kFreedesktopPortalBackground[] = "org.freedesktop.portal.Background";
+
 namespace {
 
 const char kXdgSettings[] = "xdg-settings";
 const char kXdgSettingsDefaultSchemeHandler[] = "default-url-scheme-handler";
 
+scoped_refptr<dbus::Bus> bus_;
+raw_ptr<dbus::ObjectProxy> dbus_proxy_ = nullptr;
+raw_ptr<dbus::ObjectProxy> object_proxy_ = nullptr;
```

Comment:
I think this is what @codebytere meant by saying "these" in the line above, but just to be explicit -- all three of these fields are unused & can be removed

Reply:
Must have been left behind from debugging, removed!